src/model/application/mod.rs

```diff
@@ -49,39 +49,52 @@ pub struct CurrentApplicationInfo {
     pub bot_public: bool,
     pub bot_require_code_grant: bool,
     #[serde(default)]
-    pub terms_of_service_url: Option<FixedString>,
+    pub bot: User,
     #[serde(default)]
-    pub privacy_policy_url: Option<FixedString>,
-    pub owner: Option<User>,
-    // omitted `summary` because it deprecated
-    pub verify_key: FixedString,
+    pub terms_of_service_url: Option<String>,
+    #[serde(default)]
+    pub privacy_policy_url: Option<String>,
+    #[serde(default)]
+    pub owner: User,
+    pub verify_key: String,
     pub team: Option<Team>,
     #[serde(default)]
     pub guild_id: Option<GuildId>,
     #[serde(default)]
+    pub guild: Option<PartialGuild>,
+    #[serde(default)]
     pub primary_sku_id: Option<SkuId>,
     #[serde(default)]
-    pub slug: Option<FixedString>,
+    pub slug: String,
     #[serde(default)]
-    pub cover_image: Option<FixedString>,
+    pub cover_image: String,
     #[serde(default)]
     pub flags: Option<ApplicationFlags>,
     #[serde(default)]
-    pub tags: Option<Vec<String>>,
+    pub approximate_guild_count: Option<u32>,
     #[serde(default)]
-    pub install_params: Option<InstallParams>,
+    pub approximate_user_install_count: Option<u32>,
+    #[serde(default)]
+    pub redirect_uris: Vec<String>,
     #[serde(default)]
-    pub custom_install_url: Option<FixedString>,
+    pub interactions_endpoint_url: Option<String>,
     /// The application's role connection verification entry point, which when configured will
     /// render the app as a verification method in the guild role verification configuration.
-    pub role_connections_verification_url: Option<FixedString>,
+    #[serde(default)]
+    pub role_connections_verification_url: Option<String>,
+    #[serde(default)]
+    pub event_webhooks_url: Option<String>,
+    pub event_webhook_status: EventWebhookStatus,
+    #[serde(default)]
+    pub event_webhook_type: Vec<EventWebhookType>,
+    #[serde(default)]
+    pub tags: Option<Vec<String>>,
+    #[serde(default)]
+    pub install_params: Option<InstallParams>,
     #[serde(default)]
     pub integration_types_config: HashMap<InstallationContext, InstallationContextConfig>,
-    pub approximate_guild_count: Option<u32>,
-    pub approximate_user_install_count: Option<u32>,
-    pub guild: Option<PartialGuild>,
-    pub redirect_uris: Option<Vec<String>>,
-    pub interactions_endpoint_url: Option<String>,
+    #[serde(default)]
+    pub custom_install_url: String,
```

Comment:
You have replaced a bunch of `FixedString` here with `String`, I'm assuming it's just a leftover from when this PR targeted `current`, can you change it back to use `FixedString` (for performance reasons).

Reply:
Yep, they were leftovers. I've changed them back to `FixedString` now.